Xamarin.Forms.Core/AppThemeColor.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	public class AppThemeColor : BindableObject, IDisposable
```

Comment:
this definitely doesn't look like a BindableObject. It's never parented, so it probably never gets a BindingContext set...

Reply:
Because I'm using bindable properties? Should I be using something different?